lib/eventRouter.ts

```diff
@@ -131,12 +131,12 @@ const getHandlersForLifewayEvent = (routerRules: any, event: any) => {
 
 const handleResults = (results: any, events: any) => {
   if (results.length === 0) {
-    const msg = `${logPrefix} No handlers triggered for events [ ${events.map((event: any) => event.eventType).join(' | ')} ]`;
+    const msg = `${logPrefix} No handlers triggered for event types: ${JSON.stringify(events.map((event: any) => event.eventType))}`;
```

Comment:
If you use `JSON.stringify(thing, null, 2)` it will make it formatted better.

Eg:
```
> const x = { foo: { bar: 'yolo'  } };

> console.log(JSON.stringify(x))
{"foo":{"bar":"yolo"}}

> console.log(JSON.stringify(x, null, 2))
{
  "foo": {
    "bar": "yolo"
  }
}
```